build/ci-pipeline.yml

```diff
@@ -143,6 +143,13 @@ stages:
       version: R5
       keyVaultName: $(DeploymentEnvironmentNameR5)
 
+- stage: cleanStorageAccounts
+  displayName: 'Clean Storage Accounts'
+  jobs:
```

Comment:
Nit: you could add 'dependsOn: []' so that this runs every time the pipeline runs. Since it is only deleting data over a day old it shouldn't affect currently running tests.

Reply:
Updated